From 23b16e736f6e48b66c4cbd98810723304210dcf0 Mon Sep 17 00:00:00 2001
From: port-worker <port-worker@localhost>
Date: Tue, 11 Aug 2026 08:49:20 +0000
Subject: [PATCH] build: Integrate prop modifications (2/2)

[ported to A16: build/core/main_version.mk no longer exists; version props moved to
config/version.mk as PRODUCT_PRODUCT_PROPERTIES. Drop ro.lineage.version and
ro.lineage.display.version here (ro.modversion is not set by A16 vendor/lineage
at all) so the buildinfo.sh values from part 1/2 win.]

Change-Id: I076973f902ab20011964e50955e4326c18d5b34e
---
 config/version.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/version.mk b/config/version.mk
index 7b7e646e..931deca5 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -38,8 +38,8 @@ LINEAGE_VERSION := $(PRODUCT_VERSION_MAJOR).$(PRODUCT_VERSION_MINOR)-$(LINEAGE_V
 LINEAGE_DISPLAY_VERSION := $(PRODUCT_VERSION_MAJOR)-$(LINEAGE_VERSION_SUFFIX)
 
 # LineageOS version properties
+# Do not set ro.lineage.version / ro.lineage.display.version / ro.modversion here;
+# they are set via buildinfo.sh (build/make) instead
 PRODUCT_PRODUCT_PROPERTIES += \
-    ro.lineage.version=$(LINEAGE_VERSION) \
-    ro.lineage.display.version=$(LINEAGE_DISPLAY_VERSION) \
     ro.lineage.build.version=$(PRODUCT_VERSION_MAJOR).$(PRODUCT_VERSION_MINOR) \
     ro.lineage.releasetype=$(LINEAGE_BUILDTYPE)
-- 
2.43.0

